.github/workflows/tag.yml

```diff
@@ -22,5 +22,6 @@ jobs:
         uses: goreleaser/goreleaser-action@v2.9.1
         with:
           args: release --rm-dist
+          workdir: ./cmd/celestia-appd
```

Comment:
Shouldn't this be `./cmd/`, since `celestia-appd` is a file and not a directory?

Reply:
@nicoelzer bump, I have the same question

Reply:
This actually sets it to `./cmd/celestia-appd/` (the directory)!